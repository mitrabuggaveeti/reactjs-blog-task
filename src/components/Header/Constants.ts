export const TITLE = 'EPAM BLOG';