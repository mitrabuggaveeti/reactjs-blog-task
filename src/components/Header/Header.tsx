import styles from './header.module.css';

const Header : React.FC = () => {
    return (
        <div className={styles.head} data-testid="header">
            <span>EPAM BLOG</span>
        </div>
    )
}

export default Header
