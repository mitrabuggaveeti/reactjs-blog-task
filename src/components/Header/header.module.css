.head {
    display: flex;
    align-items: center;
    justify-content: center;
    height: 80px;
    background: teal;
    color: lightcyan;
    font-size: xx-large;
}