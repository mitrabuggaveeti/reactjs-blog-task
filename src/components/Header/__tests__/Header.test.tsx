import { cleanup, render } from "@testing-library/react";
import Header from "../Header";
import renderer from 'react-test-renderer';


describe('Header component',() =>{
    afterEach(cleanup);

    it('should render', () =>{
        const { getByTestId} = render(<Header />);
        const header = getByTestId('header');
        expect(header).toBeTruthy();
    })

    it('should match snapshat', () => {
        const component = renderer.create(<Header />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})