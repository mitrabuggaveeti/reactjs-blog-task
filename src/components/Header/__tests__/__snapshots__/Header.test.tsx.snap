// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshat 1`] = `
<div
  className="head"
  data-testid="header"
>
  <span>
    EPAM BLOG
  </span>
</div>
`;
